server/src/routes/cars.ts

```diff
@@ -12,4 +17,6 @@ carsRouter.get('/', ExpressWrapper(getCarsById));
 carsRouter.get('/dashboard', authMiddleware('admin'), ExpressWrapper(getCarsDetails));
 carsRouter.delete('/:id', authMiddleware('admin'), ExpressWrapper(deleteCarsById));
 carsRouter.put('/:id', authMiddleware('admin'), ExpressWrapper(updateCar));
+carsRouter.post('/sell-car', authMiddleware('user'), ExpressWrapper(addCar));
```

Comment:
```suggestion
carsRouter.post('/', authMiddleware('user'), ExpressWrapper(addCar));
```
it's not a good practice to have a resource name that include the method,
request method `post` will definitely means it's add